plugin/openassetio_manager_bal/BasicAssetLibraryInterface.py

```diff
@@ -322,13 +328,31 @@ def getWithRelationshipsPaged(
             except Exception as exc:  # pylint: disable=broad-except
                 self.__handle_exception(exc, idx, errorCallback)
 
+    @staticmethod
+    def __parse_entity_ref(entity_ref: str) -> bal.EntityInfo:
+        """
+        Decomposes an entity reference into bal fields.
+        """
+        uri_parts = urlparse(entity_ref)
```

Comment:
[plan][minor] Would be good to have an issue to support `netloc`

Reply:
https://github.com/OpenAssetIO/OpenAssetIO-Manager-BAL/issues/60